src/fs-plus.coffee

```diff
@@ -287,7 +283,7 @@ fsPlus =
   # Public: Create a directory at the specified path including any missing
   # parent directories asynchronously.
   makeTree: (directoryPath, callback) ->
-    fs.exists directoryPath, (exists) ->
+    fsPlus.isDirectory directoryPath, (exists) ->
```

Comment:
> what happens if the path is a file?

It should fail now with an error from `mkdirp` right? Mind adding a new spec for this and seeing what happens?